v2/controllers/pod_watcher.go

```diff
@@ -182,12 +182,13 @@ OUTER2:
 	return nil
 }
 
-func (r *podWatcher) delPod(n types.NamespacedName) error {
+func (r *podWatcher) delPod(n types.NamespacedName, logger logr.Logger) error {
 	r.mu.Lock()
 	defer r.mu.Unlock()
 
 	key := n.Namespace + "/" + n.Name
 	for _, ip := range r.podAddrs[key] {
+		logger.Info("delete peer", "caller", "delPod", "pod", key, "ip", ip.String())
```

Comment:
Please log key as well.

Reply:
This logger is configured to output namespace and name.
https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/controller/controller.go#L118